Octokit/Clients/EventsClient.cs

```diff
@@ -80,7 +80,44 @@ public Task<IReadOnlyList<Activity>> GetAllForRepository(string owner, string na
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
             Ensure.ArgumentNotNull(options, "options");
 
-            return ApiConnection.GetAll<Activity>(ApiUrls.IssuesEvents(owner, name),options);
+            return ApiConnection.GetAll<Activity>(ApiUrls.Events(owner, name), options);
```

Comment:
I think the only issue I have with this PR is that we've changed the behaviour of the existing `GetAllForRepository` method. This is fine, we just need to call it out in the release notes.